src/main/java/com/redhat/devtools/lsp4ij/features/completion/LSPTypedHandlerDelegate.java

```diff
@@ -39,18 +39,32 @@ public Result checkAutoPopup(char charTyped, @NotNull Project project, @NotNull
             // The file is not associated to a language server.
             return Result.CONTINUE;
         }
+
+        Result result = Result.CONTINUE;
         if (hasLanguageServerSupportingTriggerCharacters(charTyped, project, file)) {
             // The current typed character is defined as "completion trigger characters"
             // in a associated language server, open the completion.
             AutoPopupController.getInstance(project).scheduleAutoPopup(editor);
-            return Result.STOP;
+            result = Result.STOP;
+        }
+
+        if (hasLanguageServerSupportingSignatureTriggerCharacters(charTyped, project, file)) {

```

Comment:
Remove this code block to have like before just `hasLanguageServerSupportingTriggerCharacters` and update this method like https://github.com/redhat-developer/lsp4ij/pull/467/files#r1726930342

Reply:
Sorry with my wrong comment